src/__test__/missions.test.js

```diff
@@ -0,0 +1,21 @@
+import { Provider } from 'react-redux';
+import { BrowserRouter } from 'react-router-dom';
+import renderer from 'react-test-renderer';
+import Mission from '../component/missions';
+import store from '../redux/store';
+import '@testing-library/jest-dom/extend-expect';
+
+describe('Mission component', () => {
+  it('renders correctly', () => {
+    const tree = renderer
+      .create(
+        <Provider store={store}>
+          <BrowserRouter>
+            <Mission />
```

Comment:
## [OPTIONAL]

- [ ] Kiindy make sure you mock the props that are required to render the MIsson component. currently, your test is throwing some warnings.

![image](https://user-images.githubusercontent.com/65068771/233456312-c77053f8-4ad1-4da8-9440-ed6bd00e8cc2.png)